com.bitwig.BitwigStudio.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.bitwig.BitwigStudio</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>BitwigStudio</name>
+  <summary>BitwigStudio digital audio workstation</summary>
+  <description>
+    <p>
+      BitwigStudio is an application to produce music. As a digital
+      audio workstation (DAW) it supports VST and VST3 audio plugins.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.bitwig.com/media/image/bitwig-home_bws3-2_customized-workflow/Bitwig-Home_BWS3-2_Customized-Workflow.png</image>
+      <caption>Bitwig main workflow</caption>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1" />
```

Comment:
I've no idea what this one is.

Reply:
It the content rating. Given that this is just a DAW and not a game, it's just "all"

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-content_rating

Reply:
Thank you :+1: